apps/sms/js/threads.js

```diff
@@ -18,6 +18,10 @@
     this.messages = new Map();
   }
 
+  Thread.prototype.getDraft = function() {
```

Comment:
Yep, I just made it in the simplest way that came to my mind :) And yes, it's just first step in our draft-improvements